### ISSUE TITLE
Access List section - "Allow subdomains" checkbox does not work

### ISSUE THREAD

**Issue by jamesjhedley** [blackberry/webworks-gui#202]
-open config page
-add whitelist for some random URL (i.e. http://www.google.ca)
-click "add access"

_access list entry added successfully_

-now check the "allow subdomains" check box
-save

Expected
-changes are saved and not lost

Actual
-changes are lost

**Comment by rcruz** [comment]
The checkbox on existing entries should not be editable.

**Comment by jamesjhedley** [comment]
Might actually be useful to let it remain editable, otherwise the user has to re-enter the entire URL just to allow sub domains.

**Comment by rcruz** [comment]
The user has to re-enter the URl to change the URL. We don't currently have support for editing access entries, but it can always be implemented later.

**Comment by rcruz** [comment]
Checkboxes cannot be set to readonly, but they can be disabled. Being disabled fades the checkbox, which makes it hard to see on some browsers. An image of a checked box can be used instead. Recommending this change to be in a future release.

**Comment by jamesjhedley** [comment]
Agreed. We shouldn't be using a checkbox if you can't actually set the value. An image or some text that shows allow subdomains has been set would be best.
